config.yaml

```diff
@@ -17,3 +17,8 @@ options:
       The hostname should NOT contain the LDAP scheme (e.g. ldap://) and port.
     default: "ldap.glauth.com"
     type: string
+  starttls_enabled:
+    description: |
+      Enable the StartTLS support or not.
+    default: true
```

Comment:
**nitpick/personal preference (ignore if you disagree)**: Perhaps this would be better? Looks scarier and declares intent (perhaps `starttls_disabled` would be a better name)
```suggestion
  disable_starttls:
    description: |
      Disable starttls, DO NOT USE THIS IN PRODUCTION 
    default: false
```

Reply:
Yeah, understand your point. But I would like to keep it that way if possible since users will not do a `NOT` operation in their heads. (or maybe just me doing that operation 😅)